script/openqa-clone-custom-git-refspec

```diff
@@ -15,7 +15,7 @@ if [[ -n "$GITHUB_TOKEN" ]]; then
     echo "Github oauth token provided, peforming authenticated requests"
 fi
 
-curl_github="${curl_github:-"curl${AUTHENTICATED_REQUEST}"}"
+curl_github="${curl_github:-"curl ${AUTHENTICATED_REQUEST}"}"
```

Comment:
Hm, don't know how I messed this up but what I wanted to prevent is the additional space when there is no token provided. Can you try to fix that? Also, we have a test module now for this script. Should be simple enough to add a test there as well :)

Reply:
Fixed.